tests/cypress/tests/e2e/mcpar/dashboard.feature

```diff
@@ -0,0 +1,84 @@
+Feature: MCPAR Dashboard Page - Program Creation/Editing/Archiving
+
+    Scenario: State users can create programs
```

Comment:
When you have `Examples` you should use `Scenario Outline`
I didn't even know you could get away with using Scenarios 😄